addons/stock/populate/product.py

```diff
@@ -0,0 +1,10 @@
+from odoo import models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    def _populate_get_types(self):
```

Comment:
Instead of create a custom method for this, maybe it will be cleaner to add a other generator which will override the first one in the _populate_factories:
```
res = super()._populate_factories()
res.append(("type", populate.iterate(["consu", "service", "product"], [0.3, 0.2, 0.5])))
return res
```
(done in https://github.com/odoo-dev/odoo/commits/master-logistics-populate-ryv )

Reply:
If another module adds a product type, with such a dedicated method, we can populate all types.
If you use an override for this, you will only populate the basic types + the latest type added ;).

I know that fundamentally, we only have 3 product types in Odoo atm, and thus both solutions are kinda equal, but I prefer a more generic solution.